app/Decsys/Models/Wordlist/WordType.cs

```diff
@@ -0,0 +1,7 @@
+namespace Decsys.Models.Wordlist;
+
+public enum WordType
+{
+    noun,
+    adjective
```

Comment:
```suggestion
    Noun,
    Adjective
```
⛏️ This looks more in line with the C# style guide

Reply:
To keep consistent with how the database saves "noun" and "adjectives" in other properties I've decided to go with this